barchart/services/dataService.js

```diff
@@ -1,57 +1,61 @@
-(function(){
+(function () {
     angular.module('myApp')
-    
-    .factory('dataService', 
-    function($http, $moment){
-        
+
+    .factory('dataService',
+    function ($http, $moment) {
+
         return {
             getTweets: getTweets
         }
-        
-        function getTweets(queryType, queryTerm, dateFrom, dateTo, count){
-            
+
+        function getTweets(queryType, queryTerm, dateFrom, dateTo, count) {
```

Comment:
Opening brace should be on a new line

*PHPCodeSnifferBear, severity NORMAL, section `js`.*